packages/data-point/lib/__snapshots__/index.test.js.snap

```diff
@@ -0,0 +1,20 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`lib It should explicity expose set of methods 1`] = `
+Object {
+  "create": [Function],
+  "createEntity": [Function],
+  "createReducerResolver": [Function],
+  "helpers": Object {
+    "assign": [Function],
+    "filter": [Function],
+    "find": [Function],
+    "map": [Function],
+    "omit": [Function],
+    "pick": [Function],
```

Comment:
should we expose omit and pick or just use them internally?

Reply:
sorry didn't see your message here, I think its ok, but we can def remove them from there on another iteration